core/src/main/java/org/apache/accumulo/core/client/admin/CompactionConfigurerConfig.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.accumulo.core.client.admin;
+
+/**
+ * @since 2.1.0
+ */
+public class CompactionConfigurerConfig extends PluginConfig<CompactionConfigurerConfig> {
```

Comment:
Something about this name leaves a little to be desired. I'd like to make a suggestion, but want to ask is this meant to be a configuration of a configurer?

Reply:
> Something about this name leaves a little to be desired.

No doubt, I don't like the name either.  CompactionConfigurer is a server side class that allows dynamic overriding of file creation options based on the input files. This is a client side class for configuring a CompactionConfigurer to run as part of a user initiated compaction.

Reply:
This class was deleted in b764df9 and only PluginConfig remains for use on the client side to configure a plugin.